docs/framework/debug-trace-profile/jitcompilationstart-mda.md

```diff
@@ -10,22 +10,23 @@ helpviewer_keywords:
 ms.assetid: 5ffd2857-d0ba-4342-9824-9ffe04ec135d
 ---
 # jitCompilationStart MDA
+
 The `jitCompilationStart` managed debugging assistant (MDA) is activated to report when the just-in-time (JIT) compiler starts to compile a function.  
   
 ## Symptoms  
  The working set size increases for a program that is already in native image format because mscorjit.dll is loaded into the process.  
   
 ## Cause  
- Not all the assemblies the program depends on have been generated into native format, or those that have are not registered correctly.  
-  
+ Not all the assemblies the program depends on have been generated into native format. Or, those assemblies that have been generated into native format aren't registered correctly.  
```

Comment:
```suggestion
 Not all the assemblies the program depends on have been generated into native format. Or those assemblies that have been generated into native format aren't registered correctly.  
```

Reply:
Without the comma, it seems like a sentence fragment to me. Is it correct to write a sentence like that?

Reply:
Here's one article about it https://lifehacker.com/you-can-start-a-sentence-with-and-but-or-or-1832984839

Reply:
Interesting, thanks. It still read weirdly to me, so I changed the format to what the API docs use for exceptions. Does that look okay?

Reply:
Yes, that's fine. 😄